compose-method/README.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: pattern
+title: Compose-method
+folder: compose-method
+permalink: /patterns/compose-method/
+categories: Structural
+tags:
+ - Gang of Four
```

Comment:
It's not in GoF book. I suggest using `Decoupling` tag. https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags

Reply:
Since this pattern is a small one, I would use `Idiom` in the categories field